Sources/StreamChatUI/ChatMessageList/RETHINK/MessageCell.swift

```diff
@@ -176,15 +183,21 @@ class MessageCell<ExtraData: ExtraDataTypes>: _CollectionViewCell, UIConfigProvi
         quotedMessageView?.isVisible = layoutOptions?.contains(.quotedMessage) ?? false
         quotedMessageView?.message = content?.quotedMessage
 
-        // Photo preview
-        photoPreviewView?.isVisible = layoutOptions?.contains(.photoPreview) ?? false
-        photoPreviewView?.content = content.map {
+        let attachments: _ChatMessageAttachmentListViewData<ExtraData>? = content.map {
             .init(
                 attachments: $0.attachments.compactMap { $0 as? ChatMessageDefaultAttachment },
                 didTapOnAttachment: nil,
                 didTapOnAttachmentAction: nil
             )
         }
+
+        // Photo preview
+        photoPreviewView?.isVisible = layoutOptions?.contains(.photoPreview) ?? false
```

Comment:
why do we need to do this? If the preview is not in layout options it won't be visible 🤷‍♂️